.circleci/config.yml

```diff
@@ -0,0 +1,115 @@
+####################
+# CircleCI configuration reference:
+#   https://circleci.com/docs/2.0/configuration-reference
+####################
+# CircleCI built-in environment variables:
+#   https://circleci.com/docs/2.0/env-vars/#built-in-environment-variables
+####################
+
+
+####################
+# Templates: see "anchors" in https://learnxinyminutes.com/docs/yaml/
+####################
+
+save_cache_settings: &save_cache_settings
+  key: python_mozetl-{{ .Branch }}-{{ .Environment.CIRCLE_WORKFLOW_ID }}
+  paths:
+    - ~/python_mozetl/.tox
+
+prep_cache_settings: &prep_cache_settings
+  name: Clean directories to restore
+  command: |
+    : # no-op
```

Comment:
We can remove this and its usages is there's nothing to do here. I created `prep_cache_settings` for t-b-v because the directories already existed in the docker image we were starting from. Looks like that's not a complication here.